test/vault/Vault.t.sol

```diff
@@ -82,6 +84,9 @@ contract VaultTest is Test, NoIsolate, GasSnapshot {
 
     function test_bytecodeSize() public {
         snapSize("VaultBytecodeSize", address(vault));
+        if (address(vault).code.length > 24576) {
```

Comment:
Vault is very small, should never excess the limit , but it is fine to add this.